tests/flow/test_tdigest.py

```diff
@@ -475,10 +475,16 @@ def test_tdigest_cdf(self):
             self.assertOk(self.cmd("tdigest.add", "tdigest", x, 1.0))
 
         self.assertAlmostEqual(
-            0.01, float(self.cmd("tdigest.cdf", "tdigest", 1.0)), 0.01
+            0.01, float(self.cmd("tdigest.cdf", "tdigest", 1.0)[0]), 0.01
         )
         self.assertAlmostEqual(
-            0.99, float(self.cmd("tdigest.cdf", "tdigest", 99.0)), 0.01
+            0.99, float(self.cmd("tdigest.cdf", "tdigest", 99.0)[0]), 0.01
+        )
+        self.assertAlmostEqual(
+            0.99, float(self.cmd("tdigest.cdf", "tdigest", 1.0, 99.0)[1]), 0.01
+        )
+        self.assertAlmostEqual(
+            0.01, float(self.cmd("tdigest.cdf", "tdigest", 99.0, 1.0)[1]), 0.01
         )
 
     def test_negative_tdigest_cdf(self):
```

Comment:
@ashtul can we add a negative test for parsing errors in the middle of the command parsing?